src/pages/todos/components/TodoList/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Dispatch, SetStateAction, useCallback } from 'react';
+
+import { TodoType } from '@pages/todos';
+import {
+  StyledTodoItem,
+  StyledTodoListItems,
+} from '@pages/todos/components/TodoList/TodoList.styles';
+import CheckBoxComponent from '@components/CheckBox';
+import ButtonComponent from '@components/Button';
+
+type TodoListProps = {
+  todos: TodoType[];
+  setTodos: Dispatch<SetStateAction<TodoType[]>>;
+};
+
+function TodoListComponent({ todos, setTodos }: TodoListProps): React.ReactElement {
+  const handleChangeStatus = useCallback(
+    (id: string) => {
+      setTodos((previousTodos) =>
+        previousTodos.map((todo_) =>
+          id === todo_.id
+            ? {
+                ...todo_,
+                isDone: !todo_.isDone,
+              }
+            : todo_
+        )
+      );
+    },
+    [setTodos]
+  );
+
+  const handleDeleteTodo = useCallback(
+    (id: string) => {
+      setTodos((previousTodos) => previousTodos.filter((todo_) => id !== todo_.id));
```

Comment:
```suggestion
      setTodos((previousTodos) => previousTodos.filter((todo) => id !== todo.id));
```
this must be left over too